### PR TITLE
Configured routes, model, and migration for Stops table.  Seeded with…

### PR DESCRIPTION
**Pull request by jarmstro0**
… initial data.

### CODE REVIEW

**Comment by jarmstro0** [comment]
Ready to merge with master branch

File: app/models/stop.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class Stop < ApplicationRecord
```
Comment (CassandraNewell) [inline app/models/stop.rb:1]: You need to create a test for this new model

File: config/routes.rb [hunk 2]
```diff
@@ -1,5 +1,6 @@
 Rails.application.routes.draw do
```
Comment (CassandraNewell) [inline config/routes.rb:1]: A PR must contain one new _feature_ and whatever code is needed to implement the feature. A feature is described as a new functionality for a user -- "the user can now view X" or whatever -- and will inevitably involve creating/modifying the frontend.

**Review by CassandraNewell — CHANGES_REQUESTED** [review]

File: config/routes.rb (hunk 2 above)
Comment (sromelus) [reply config/routes.rb:1]: User features added, show an index page

File: app/models/stop.rb (hunk 1 above)
Comment (sromelus) [reply app/models/stop.rb:1]: test created

**Review by CassandraNewell — APPROVED** [review]

**Pull request merged**